CHANGES.md

```diff
@@ -75,6 +75,7 @@
 
 * Support for X source added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 * Added ability to use JdbcIO.Write.withResults without statement and preparedStatementSetter. ([BEAM-12511](https://issues.apache.org/jira/browse/BEAM-12511))
+- Added ability to register URI schemes to use the S3 protocol via FileIO. ([BEAM-12354](https://issues.apache.org/jira/brows/BEAM-12354]).
```

Comment:
nit: Wrong Jira ID (`BEAM-12435`) and URL (`https://issues.apache.org/jira/browse/BEAM-12435`) here.

Reply:
Good catch! Sorry about that.